examples/pytorch/language-modeling/run_clm.py

```diff
@@ -308,6 +366,10 @@ def main():
     # Set seed before initializing model.
     set_seed(training_args.seed)
 
+    server = xp.start_server(9012)
+    logger.info('Profiling server started: {str(server)}')
```

Comment:
My bad, but a small bug here - this should be an f-string: `f'Profiling.. {str(server)}'`